app/editor/src/components/tool-bar/ToolBarSection2.tsx

```diff
@@ -0,0 +1,21 @@
+import { Row } from 'tno-core';
+import * as styled from './styled';
+
+export interface IToolBarSection2Props {
+  label: string;
```

Comment:
Maybe make this optional so that we can use it as a title section in other parts of the application if required. And then `title: string;` will be a required prop to determine the heading for the given section.